source/documentation/guidance/pentest.md

```diff
@@ -1,17 +1,14 @@
 ## Penetration testing
 
-You must penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform.
+You [must](#responsibility-model) penetration test your app to make sure it’s secure. Note that you can only penetration test your app; you cannot penetration test the platform. 
 
-Because GOV.UK PaaS runs on Amazon Web Services (AWS), you need permission from AWS to penetration test your app. The PaaS team will request permission for you because the process requires root credentials for the AWS portal.
-
-Send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) and the PaaS team will share it with AWS.
+GOV.UK PaaS runs on Amazon Web Services (AWS), but there is no need for us to inform AWS when you carry out penetration tests. However, you should still send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) so that we are aware of it and can approve it.
```

Comment:
Perhaps we could shorten this to focus on what the user must do, rather than what they don't need to do:

```
You must send the following information to [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) so that we are aware of your test, and can approve it.
```